openrtb_ext/imp.go

```diff
@@ -42,6 +42,16 @@ type ExtImpPrebid struct {
 	Options *Options `json:"options,omitempty"`
 
 	Passthrough json.RawMessage `json:"passthrough,omitempty"`
+
+	Floors *ExtImpPrebidFloors `json:"floors,omitempty"`
+}
+
+type ExtImpPrebidFloors struct {
+	FloorRule      string  `json:"floorRule,omitempty"`
+	FloorRuleValue float64 `json:"floorRuleValue,omitempty"`
+	FloorValue     float64 `json:"floorValue,omitempty"`
+	FloorMin       float64 `json:"floorMin,omitempty"`
+	FloorMinCur    string  `json:"floorMinCur,omitempty"`
```

Comment:
IMHO: Consider dropping the "Floor" prefix for every variable name (keeping the json as needed). That's a lot of repetition.